catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java

```diff
@@ -203,6 +223,22 @@ private String parseSrc(String localSource) {
     return src;
   }
 
+  private void parseSrcs(String localSource) {
+    for (int i = 0; i < srcs.size(); i++) {
```

Comment:
✏️ This method could be replaced with `Collections.replaceAll`.

https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#replaceAll-java.util.List-T-T-

Reply:
Since we are doing case insensitive comparison, it doesn't look like this will work